app/views/report/_checked.haml

```diff
@@ -0,0 +1,40 @@
+%table.table
+  %tr
+    %th
+      = t("report.created_at")
+    %th
+      = t("report.reported_by")
+    %th
+      = t("report.reason")
+    %th
+      = t("report.type")
+    %th
+      = t("report.author")
+    %th
+      = t("report.decision")
+    %th
+      = t("report.action")
+  - @reviewed_reports.each do |report|
+    %tr
+      %td
+        = I18n.l(report.created_at, format: :short)
+      %td
+        = link_to(report.user.username, user_profile_path(report.user.username))
+      %td
+        = report.text
+      %td
+        = report.item_type
+      %td
+        - unless report.originator_diaspora_handle.nil?
+          = link_to(report.originator_diaspora_handle, "/people/#{report.originator_guid}")
```

Comment:
If we don't know the author of the reported item anymore (because it was deleted before the migration), then I would show "Unknown" here instead of just blank.